docs/administration/modules/ROOT/pages/checkpointing.adoc

```diff
@@ -84,3 +84,17 @@ EDN::
 
 == `FileSystem` Checkpoint Store parameters
 * `path` (required, string/`File`/`Path`/`URI`): path to store checkpoints.
+
+=== Note about using `FileSystem` Checkpoint Store
+
+Most Cloud providers offer some high-performance block storage facility (e.g.
```

Comment:
[adoc convention](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) is one sentence per line rather than wrapping - I tend to then use `visual-line-mode` in emacs to soft-wrap them again.